docs/measures/pid.rst

```diff
@@ -413,3 +413,31 @@ They then define two partial information lattices, one quantified locally by :ma
    ║  {1}   │ 0.5000 │ 0.5000 ║
    ║ {0}{1} │ 0.0000 │ 0.0000 ║
    ╚════════╧════════╧════════╝
+
+
+
+---------------------
+
+Taking a functional perspective as in \IGK, \I[RAV] defines a bivariate redundancy as the maximum coinformation between two sources :math:`X_0, X_1', a target :math:`Y`, and a deterministic function of the inputs :math:`f(X_0,X_1)`.
```

Comment:
\I[RAV] means nothing. add a macro to docs/_static/dit_mathjax.js and dos/conf.py (look for the latex macros). also, wrap the \IGK and \IRAV calls in :math:``